packages/theming/android-theme/src/createAndroidTheme.ts

```diff
@@ -0,0 +1,29 @@
+import { ThemeReference } from '@fluentui-react-native/theme';
+import { getAndroidTheme } from './androidTheme';
+import { Appearance } from 'react-native';
+import { Theme } from '@fluentui-react-native/theme-types';
+
+export type AppearanceOptions = 'light' | 'dark';
```

Comment:
Does this need to be exported? Or did you copy this from the default theme?

Reply:
yep copied the approach of default theme, though these are not needed and removed.